### PR TITLE
Update CVA5 support to latest version

### PR DESCRIPTION
**Pull request by yannickl96**
The config file and top-level interface changed, resulting in broken simulation. 
I updated the patch file and fixed the core version to a specific commit so it works out-of-the-box again. 

Main drawback is that we now have to actively bump the commit SHA in order to update the core version.

### CODE REVIEW

File: riscv/cva5/cva5_tapasco.patch [hunk 1]
```diff
@@ -11,10 +11,9 @@ index c7d5550..19d71d5 100755
 +        INCLUDE_S_MODE : 0,
 +        INCLUDE_U_MODE : 0,
          INCLUDE_MUL : 1,
--        INCLUDE_DIV : 1,
```
Comment (cahz) [inline riscv/cva5/cva5_tapasco.patch:14]: Please keep the configuration the same. Divider is not needed in most of the tapasco-riscv use cases.

**Review by cahz — CHANGES_REQUESTED** [review]

File: riscv/cva5/cva5_tapasco.patch [hunk 2]
```diff
@@ -280,7 +274,8 @@ index 0000000..0cb6a56
 +
 +	.m_axi(m_axi),
 +	.m_avalon(m_avalon),
-+	.m_wishbone(m_wishbone),
++	.dwishbone(m_wishbone),
++    .iwishbone(m_wishbone),
```
[the remark this answers is not in the document]
Comment (cahz) [reply riscv/cva5/cva5_tapasco.patch:278]: Furthermore, it is better to also rename `m_wishbone` to `i` and `d` in our wrapper.

File: riscv/cva5/cva5_tapasco.patch (hunk 1 above)
Comment (yannickl96) [reply riscv/cva5/cva5_tapasco.patch]: Done!

File: riscv/cva5/cva5_tapasco.patch (hunk 2 above)
[the remark this answers is not in the document]
Comment (yannickl96) [reply riscv/cva5/cva5_tapasco.patch]: Done!

**Pull request merged**